WalletLibrary/WalletLibrary/Networking/Entities/OpenID4VCI/CredentialOffer.swift

```diff
@@ -9,25 +9,81 @@
 struct CredentialOffer: Codable
 {
     /// The end point of the credential issuer metadata.
-    let credential_issuer: String?
+    let credential_issuer: String
     
     /// The state of the request. Opaque to the wallet.
-    let issuer_session: String?
+    let issuer_session: String
     
     /// The credential ids that will be used to issue the Verified ID.
-    let credential_configuration_ids: [String]?
+    let credential_configuration_ids: [String]
```

Comment:
Are these fields mandatory in standards since they are non-nulls now?

Reply:
they are mandatory, and we do the validation logic differently for the CredentialOffer so we can make class's properties mandatory. If we were using JSONDecoder to parse this data from Data from network call to this model, we would want to make all of these properties nullable in case there is a parsing issue. But this data model is one layer above network call and is parsed in the Handler so they can be non-nullable.